cloud-sql/postgres/knex/test/createTable.test.js

```diff
@@ -43,7 +53,7 @@ before(async () => {
 
 it('should create a table', (done) => {
   exec(
-    `node createTable.js ${DB_USER} ${DB_PASS} ${DB_NAME} ${CONNECTION_NAME}`,
+    `node createTable.js ${DB_USER} ${DB_PASS} ${DB_NAME} ${CONNECTION_NAME} ${DB_HOST}`,
```

Comment:
Is there a plan to eventually move away from the CLI mechanism?

Reply:
The MySQL and SQL Server samples have an `ensureSchema` method that creates a table if it doesn't exist each time `server.js` is run. Should we switch to using that approach here?

Reply:
I think that would be a good idea but it seems lower priority. Probably something for you/your team to decide.

Reply:
+1 - probably best in a separate PR.